lib/graphql/query.rb

```diff
@@ -148,7 +149,9 @@ def query_string
       @query_string ||= (document ? document.to_query_string : nil)
     end
 
-    def_delegators :@schema, :interpreter?
+    def interpreter?
+      @interpreter
+    end
```

Comment:
Apparently Forwardable makes an allocation in these calls, this fixed that (and made them faster)